src/internal/packager/git/pull.go

```diff
@@ -119,3 +118,45 @@ func (g *Git) pull(gitURL, targetFolder string, repoName string) {
 		}
 	}
 }
+
+func (g *Git) pull() error {
```

Comment:
Almost wonder if this should be separated into an actual pull.go and then rename this file to something like download.go or retrieve.go since it is driving other git actions.

Reply:
I'm not opposed to that at all